config/routes.rb

```diff
@@ -77,7 +77,11 @@
         json_api_resources :pages, controller: "project_pages"
       end
 
-      json_api_resources :workflows, links: [:subject_sets], versioned: true
+      json_api_resources :workflows, links: [:subject_sets], versioned: true do
+        member do
+          post :retire_subject
```

Comment:
I like that. The inverse might also be an idea: `/subjects/1/links/retired_workflows`, since that's how it is stored in the db?